internal/testing/e2e/backfill_test.go

```diff
@@ -132,20 +131,39 @@ func TestBackfillFrontendLifecycle(t *testing.T) {
 	require.Nil(t, get)
 }
 
-func TestProposedBackfillCreate(t *testing.T) {
-	ctx := context.Background()
+func TestAcknowledgeBackfill(t *testing.T) {
```

Comment:
Actually, this test does not cover AcknowledgeBackfill at all, otherwise, you would notice that you do not call the Acknowledge backfill method from a statestore. This test should be completely rewritten.

Reply:
This is E2E test, so it is hard to test this.
I know that there is an additional Redis state change - this would be covered in E2E test.
It covers one of main effects of acknowledgment - all tickets get assigned.

Reply:
Ok, I see why you can not cover AcknowledgeBackfill step in e2e, you still can do that in unit test to check the redis state. What do you think?

Reply:
Added UT.